webui/react/src/index.tsx

```diff
@@ -23,13 +23,16 @@ if (process.env.PUBLIC_URL && history.location.pathname === '/') {
   history.replace(process.env.PUBLIC_URL);
 }
 
-ReactDOM.render(
-  <React.StrictMode>
-    <HistoryRouter basename={process.env.PUBLIC_URL} history={history}>
-      <App />
-    </HistoryRouter>
-  </React.StrictMode>,
-  document.getElementById('root'),
+const container = document.getElementById('root');
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const root = createRoot(container!);
+
+root.render(
+  // <React.StrictMode>
```

Comment:
I disabled `Strict Mode` for now cuz we rely on the following code.
```ts
  useEffect(() => {
    ...
    return () => canceler.abort();
  }, [canceler]);
```
It doesn't work well with Strict Mode cuz the new Strict Mode renders component twice in development mode

Reply:
Can you elaborate on this a bit more? What was strict mode complaining about?

Reply:
>This new check will automatically unmount and remount every component, whenever a component mounts for the first time, restoring the previous state on the second mount.
https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-strict-mode

New Strict Mode unmount first, and remount afterwards in development mode. Due to this, `canceler.abort();` is called unexpectedly, so API calls are aborted. 
We actually talked about this in July, and couldn't find a good solution with `canceler.abort();` and Strict Mode.

Reply:
Not really
Even if I fix this, the behavior looks like [this](https://determined-ai.slack.com/archives/CKQ3EV0G6/p1658334035443339?thread_ts=1658270891.880979&cid=CKQ3EV0G6)
I dont think its good.